usb/usb_tool.js

```diff
@@ -59,6 +67,24 @@ const ws_connect = () => {
 }
 ws_connect()
 
+usbDetect.on('add', function(usb_port) { 
+    // The usb_port object does not have the path field needed to open
+    // a connection to the port so instead check if the manufacturer
+    // is correct, and if it is find which port its on and select it.
+    // This could go wrong if 2 arduinos are plugged in
```

Comment:
Can we provide some checking to see if two potential devices are connected? I did this [previously](https://github.com/UWO-Aero-Design/gnd-station/blob/03b9f27fff5ae569a1404c29db7bba48708652b3/api/config/usb.js#L72-L85) by creating a list of candidate devices and only attempting an auto connect if the list had exactly one entry.

Reply:
I change that to